users.js

```diff
@@ -46,3 +46,36 @@ const users = [
 	{ "id":45,"name":"Mays Noel","lastly":"00:00 18.04.2017"},
 	{ "id":46,"name":"Solomon Vang","lastly":"10:13 30.11.2017"}
 ];
+
+
+const newUserArr = [];
```

Comment:
1. Skoro widzę, że to tablica, to nie nazywaj jej jeszcze dodatkowo `Arr`.
2. Nazwa newUsers byłaby lepsza, liczba mnoga, ze względu na to, że trzymamy w zmiennej kolekcję.